### PR TITLE
chroe(rpc): keep rpc doc endpoints order same as rpc.json

### LINKED ISSUE #1229: Keep the endpoints order in rpc.json when generating RPC doc
Keep the order, thus we can sort endpoints into logic groups.

### PR DESCRIPTION
**Pull request by zjhmale**
Closes #1229

### CODE REVIEW

**Comment by zjhmale** [comment]
ping @doitian

File: devtools/doc/rpc.py [hunk 1]
```diff
@@ -102,7 +102,6 @@ def main():
     print("# CKB JSON-RPC Protocols")
     newline(2)
 
-    cases = sorted(cases, key = lambda x: x["module"] + x["method"])
```
Comment (doitian) [inline devtools/doc/rpc.py]: It requires the JSON must put the RPCs in the same module together. But I think it is a reasonable requirement.

**Comment by doitian** [comment]
We have temporarily held all the PRs until #1249 is merged.

**Comment by u2** [comment]
Please rebase to develop branch.

**Comment by zjhmale** [comment]
@u2 done :)

File: devtools/doc/rpc.py (hunk 1 above)
Comment (zjhmale) [reply devtools/doc/rpc.py]: agreed

**Comment by u2** [comment]
> @u2 done :)

Some urgent mergings lead the conflict. Please rebase again, thanks.

**Pull request merged**